cli/cmd/install.go

```diff
@@ -398,34 +398,34 @@ control plane.`,
 }
 
 func installRunE(ctx context.Context, options *installOptions, stage string, flags *pflag.FlagSet) error {
-	values, _, err := options.validateAndBuild(ctx, stage, flags)
+	values, err := options.validateAndBuild(ctx, stage, flags)
 	if err != nil {
 		return err
 	}
 
 	return render(os.Stdout, values)
 }
 
-func (options *installOptions) validateAndBuild(ctx context.Context, stage string, flags *pflag.FlagSet) (*l5dcharts.Values, *pb.All, error) {
+func (options *installOptions) validateAndBuild(ctx context.Context, stage string, flags *pflag.FlagSet) (*l5dcharts.Values, error) {
```

Comment:
This had to be done, as the linter complains  that `pb.All` return is not being used anywhere